at_response.c

```diff
@@ -1229,6 +1229,8 @@ static int at_response_cmgr (struct pvt* pvt, const char * str, size_t len)
 				return 0;
 			}
 
+			ast_log(LOG_NOTICE, "[%s] Incoming message parsed '%s'\n", PVT_ID(pvt), str);
+
 			ast_debug (1, "[%s] Successfully read SMS message\n", PVT_ID(pvt));
```

Comment:
> Can cancel / close this pull requests ?

I don't mind either way. You can replace the NOTICE with an ast_verb and I'll merge.
Or you can close this PR and do it differently.

All up to you :)